examples/plot_timeseries.py

```diff
@@ -0,0 +1,32 @@
+"""
+TimeSeries Example
+==================
+
+An example to show some basic, but very important functions in Sunpy's TimeSeries.
+
+For more information about TimeSeries, http://docs.sunpy.org/en/stable/guide/data_types/timeseries.html
+For more information about AstroPy Units, http://docs.astropy.org/en/stable/units/
+"""
+
+import heliopy.data.ulysses as ulysses
+import matplotlib.pyplot as plt
+from datetime import datetime
+
+starttime = datetime(1993, 1, 1, 0, 0, 0)
+endtime = datetime(1993, 2, 1, 0, 0, 0)
+
+timeseries_data = ulysses.swics_abundances(starttime, endtime)
+
+# timeseries_data is a TimeSeries data type
+# Using the timeseries.data function, one can obtain the Pandas DataFrame of the data
```

Comment:
I think this is the most important point of using TimeSeries. Ease of Pandas DataFrame plus units!